src/wallet/rpcwallet.cpp

```diff
@@ -2660,6 +2664,104 @@ UniValue z_getnewaddress(const UniValue& params, bool fHelp)
     }
 }
 
+UniValue z_getnewaccount(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+    if (fHelp || params.size() > 0)
+        throw runtime_error(
+            "z_getnewaccount\n"
+            "\nSelects the next available sequential account number within the UA-compatible HD seed phrase,"
+            "\ngenerates the corresponding spend authorities for the best and second-best shielded pools, and the transparent pool,"
+            "\nreturns the new account number and the default / first UA containing the default addresses for each pool.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"account\": n,       (numeric) next available account number\n"
+            "  \"unifiedaddress\"    (string) The default address for this account\n"
```

Comment:
```suggestion
            "  \"unifiedAddress\"    (string) The default address for this account\n"
```

Reply:
Should this method take a list of pools for which the unified address should contain receivers? Or do we intend a zcashd-wide policy that the set is `{transparent, sapling, orchard}` (and, by implication, ensure that we do not select account identifiers that do not produce a valid Sapling address at diversifier index 0?